.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="app"/>
+	<classpathentry kind="src" path="target/scala-2.11/routes/main"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="output" path="target/scala-2.11/classes"/>
+</classpath>
```

Comment:
do not add this generated fileif you need to you can add .classpath to .gitignore and commit that